src/ci/docker/host-x86_64/mingw-check/validate-mir.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+echo $'[rust]\nvalidate-mir-opts = 3' > /tmp/validate-mir.toml
+if ! python3 ../x.py build --stage 0 library/std --config /tmp/validate-mir.toml; then
+    echo "failed validate mir!"
+    exit 1
+fi
+rm /tmp/validate-mir.toml
+echo "finished validate mir"
```

Comment:
Hmm, I'm not sure why this script and the change to `SCRIPT` were necessary? bootstrap is already doing the logic for you, you don't need to make sure to invoke it only on stage 0.

Reply:
you means add `RUST_CONFIGURE_ARGS` at here?
https://github.com/rust-lang/rust/blob/07c993eba8b76eae497e98433ae075b00f01be10/.github/workflows/ci.yml#L52

Reply:
No, in the dockerfile itself.

Reply:
updated!